### DISCUSSION TITLE
[V2] How to close a WebviewWindow in Android

### DISCUSSION THREAD

**Discussion by Burial0268** [tauri-apps/tauri#11024] [Q&A] [answered]
## Question:

I have a Tauri app for Android, and I'm currently using the `SplashScreen` feature.
In the documentation, I saw that I should use like 
`AppHandle::get_webview_window("main").unwrap().show().unwrap();`

I tried, and it works fine on Windows dev.
But when I run the `pnpm tauri build -- --target=aarch64` an error occurred:
error[E0599]: no method named `show` found for struct `tauri::WebviewWindow` in the current scope

How can I fix this bug?

---
## Info:

tauri:          2.0.0-rc.14
tauri-build: 2.0.0-rc

I can give more info if needed.

**Comment by Burial0268** [comment]
Is it because it's not available on Android?

**Reply by FabianLars** [reply, marked as answer]
yep. mobile is currently restricted to one webview/window. Things like splashscreens must be handled in your frontend for now (or in the raw android project).

**Reply by Burial0268** [reply]
Well, that sucks...... Thanks for your reply

**Reply by Burial0268** [reply]
could we just use Android's Activity to impl this? I'm not familiar with Android Development but I think this may work

**Reply by Ahmard** [reply]
@Burial0268 have you finally achieved this?
Facing this issue now as well.

**Reply by Burial0268** [reply]
frontend-only splashscreen i'm currently using.
additionally, using the message channel as signal

**Reply by Ahmard** [reply]
Doing the same too, but my Auth library is blocking UI rendering until it gets user profile response from server.
I get white screen for some ms or secs when network is slow.
